src/librustc/hir/map/definitions.rs

```diff
@@ -149,7 +105,7 @@ impl DefPathTable {
     }
 
     pub fn size(&self) -> usize {
-        self.key_to_index.len()
+        self.index_to_key.iter().map(|v| v.len()).sum()
```

Comment:
@Mark-Simulacrum is right, this is a 2-element array. And it's called only once per upstream crate IIRC. I would not worry about it.